src/_layouts/popout.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {% if jekyll.environment == 'production' and site.google_analytics %}
+  {% include google-analytics.html %}
+  {% endif %}
+
+  <style>
+    body {
+      margin: 0;
+      overflow: hidden;
+    }
+    a.noh-logo {
+      left: 2%;
+      position: absolute;
+      top: 2%;
+      z-index: 2;
+    }
+    svg, a.credits {
+      opacity: .5;
+      transition: opacity .2s ease;
+    }
+    a.credits {
+      color: white;
+      font-family: sans;
```

Comment:
Should we set this to `Source Sans pro` et al. like the site menus, or `Source serif pro` et al. like the site contents?

Reply:
I'd vote against hauling in "Source Sans Pro" from Google Fonts just for this, but you have caught the fact that I've used the wrong generic value for `font-family` (should be `sans-serif`), so a fix for that is now pushed.  Thanks!